stdlib/marshal.mli

```diff
@@ -36,9 +36,9 @@
    belong to the given type.
 
    Values of extensible variant types, for example exceptions (of
-   extensible type [exn]), returned by the unmarhsaller should not be
+   extensible type [exn]), returned by the unmarshaller should not be
    pattern-matched over through [match ... with] or [try ... with],
-   because unmarshalling does not preserve the information required for
+   because unmarshaling does not preserve the information required for
```

Comment:
Counting (in OCaml-trunk) the number of appearances of :  

' unmarshaling ' - ' marshaling ' - ' marshaled ' - ' marshal ' - 'unmarshal ' gives : 
43 + 187 + 166 + 30 + 54 = 480

' unmarshalling ' - ' marshalling ' - ' marshalled ' - ' marshall ' - 'unmarshall ' gives : 
 20 + 60 + 45 + 0 + 0 = 125

The majority wins, no ?

And we also have a shorter writing  :)